tide_site.install

```diff
@@ -42,11 +44,20 @@ function tide_site_update_10001() {
     'core.entity_view_display.taxonomy_term.sites.default',
   ];
   foreach ($form_configs as $form_config) {
-    $config = \Drupal::configFactory()->getEditable($form_config);
-    $config_read = _tide_read_config($form_config, $config_location, FALSE);
-    $config->set('dependencies', $config_read['dependencies']);
-    $config->set('content', $config_read['content']);
-    $config->set('hidden', $config_read['hidden']);
-    $config->save();
```

Comment:
Arbitrarily overwriting active configurations may lead to layout inconsistencies.